schema/misc.fbs

```diff
@@ -76,3 +76,18 @@ table ControllableTeamInfo {
   /// The bots that RLBot will allow this client to control.
   controllables: [ControllableInfo] (required);
 }
+
+/// As an interface message, this requests for a specificed agent to have its ability to render changed.
+/// This changed will then be broadcasted to all current connections as a core message.
+/// Does nothing if rendering has been completely disabled.
```

Comment:
> /// Does nothing if rendering has been completely disabled.

You mean, if `rendering_enabled=false`? Imo, it should always do something, and `rendering_enabled` is just the default value.

Reply:
Ah, right. Good point. If bots/scripts can turn their rendering on, then there must be an always-off option for tournaments.

I think that is a good proposal. I am wondering though, if Enabled is ever going to be used since you can't force bots to render anyway. Maybe a OnByDefault and OffByDefault makes more sense than Enabled and OnRequest? "Off" is also in line with the mutators options. For instance:
```c
enum DebugRendering {
  OffByDefault,  // default, alternatively just "Off"
  OnByDefault,  // alternatively just "On"
  AlwaysOff,
}
```

Reply:
I think `Off` vs `AlwaysOff` will be confusing, so I like this